.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.val.md

```diff
@@ -0,0 +1,30 @@
+## Release to VAL
+
+This Pull Request affects a release from master to val.
+
+#### Closed Issues in this release
```

Comment:
@ayoneeyee see comment on the production template.  Same thing; I vote you rip this section out.